python_scripts/linear_models_regularization.py

```diff
@@ -159,17 +159,18 @@
 # previous plot, we see that the magnitude of the weights are shrunk towards
 # zero in comparison with the linear regression model.
 #
+# Regularization will add constraints on weights of the model. We saw in the
```

Comment:
Instead of this paragraph here, I would explicitly reproduce the previous experiment and increase even more alpha and call the attention of the user to the change of the scale of the weights and conclude regarding the impact of `alpha`. Once done, we can then add that a negative alpha does not make sense then.

Reply:
```python
# %% [markdown]
#
# In the previous example, we fixed `alpha=100`. We will now check the impact of the value
# of `alpha` by increasing its value.

# %%
# Add some code here to reproduce with alpha=1e6

# %% [markdown]
#
# Looking specifically to weights values, we observe that increasing the value of `alpha` 
# will decrease the weight values. A negative value of `alpha` would actually enhance the
# large weights and promote overfitting. Thus, in practice, `alpha` is always expected to be
# a positive value.
```